nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -187,6 +187,8 @@
 
 - [OpenGFW](https://github.com/apernet/OpenGFW), an implementation of the Great Firewall on Linux. Available as [services.opengfw](#opt-services.opengfw.enable).
 
+- [Whoogle Search](https://github.com/benbusby/whoogle-search), a self-hosted, ad-free, privacy-respecting metasearch engine. Available as [services.whoogle-search](options.html#opt-services.whoogle-search.enable).
```

Comment:
Just checking, since now 25.05 is the next release: you want the new module backported?

Reply:
ah no, I've moved the entry to the 25.05 release notes.